tests/baselines/reference/contextualTypeWithTuple.types

```diff
@@ -15,7 +15,7 @@ var numStrTuple2: [number, string] = [5, "foo", true];
 
 var numStrBoolTuple: [number, string, boolean] = [5, "foo", true];
 >numStrBoolTuple : [number, string, boolean]
->[5, "foo", true] : [number, string, boolean]
+>[5, "foo", true] : [number, string, true]

```

Comment:
this was the reason we made this original change... to make the error messages look nicer. but i guess we can not differentiate between `true | false` and `boolean` in any meaningful manner.